### DISCUSSION TITLE
RTC Clock in KIP Dashboard

### DISCUSSION THREAD

**Discussion by veleirocica** [bareboat-necessities/lysmarine_gen#111] [Q&A] [answered]
I'm trying to enter local time into BBN's KIP Dashboard. I asked the Slack Signalk-Serv group and one of the developers told me that the solution would be through Node-red.
I would like to know if there is any tutorial to program a hardware clock (DS1307) in Node-Red. Thanks.

**Comment by mgrouch** [comment, marked as answer]
NodeRed would be absolute overkill for this small task. 
KIP allows embedding external URLs.
You can simply create a page with JavaScript which will display system time, and embed this page into KIP. 

And realistically this is a general KIP dashboard question. It's better to ask in KIP (SignalK Slack channel group: http://slack-invite.signalk.org/)

Thanks

**Reply by veleirocica** [reply]
If anyone wants a digital clock on the KIP dashboard, go to the link https://signalk-dev.slack.com/archives/C03F1MKQG/p1630626252082400?thread_ts=1630621403.082100&cid=C03F1MKQG
